posthog/queries/event_query/event_query.py

```diff
@@ -73,6 +77,9 @@ def __init__(
         if not self._should_join_persons:
             self._determine_should_join_persons()
 
+        if not self._should_join_sessions:
```

Comment:
Is this temporary / here as long as we're computing properties? Or is this a more permanent thing?

Asking because we've been getting rid of JOINs in persons and groups with person-on-events, but none of that work will apply to sessions like so. 

I think it's _fine_ as long as its here only for computed properties, but not if session_duration becomes a non-computed property, and we still needed this join. Asking so it helps me write subsequent code better.